### PR TITLE
[SEMI-MODULAR] [READY] Brig Physician

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Adds the Brig Physician Job. Pretty much a more modular port of https://github.com/TheSwain/Fulpstation/pull/473.
Read more on that if you're interested. All of it was ported APART from the Medical baton, I settled with just giving a normal stun baton instead (I mean, it still works for surgery, so...).

image

## Extras

- Adds a folder for overwrites.
- Adds a few readme.MD files to explain both the brigdoc module (alongside what TG files were touched) + the overwrite module (Thanks, Skyrat).
- Adds support for more jobs (with little struggle, for the most part).

## Does this cure World Cancer?

No??? Why would it?

## Why It's Good For The Game

Adds a highly requested job that a lot of the playerbase would appreciate. Besides, the more jobs, the more people that will get to play roundstart (Even if right now it's just one slot with one extra latejoin).
I meannn, who doesn't love themselves some brig doc?

Keeping this as a draft until milestone complete to avoid stale.

## Changelog
:cl:
add: Brig Physician job, don't mind the prisoners being nuggets.
/:cl:

### CODE REVIEW

**Comment by remove32** [comment]
image

**Comment by Horatio22** [comment]
I like the sprites! I think adding the red lines to the lab coat was a good idea. Hopefully should avoid some confusion that way.

**Comment by remove32** [comment]
so on the high level why is this better than HoS asking for on-hand med staff

**Comment by SgtHunk** [comment]
> so on the high level why is this better than HoS asking for on-hand med staff

Well.
1- Lateround med bay is a mess.
2- Corpses on medbay aren't well secured, and dead sec usually get looted.
3- You can't 100% trust doctors.
4- Other doctors/paramedics 90% of the time dont have access into brig to get that one prisoner who husked themselves on a shocked grille that was on a vacuum.

Brig Physician:
- Can't be antag.
- Has all the tools required both in Infirmary + roundstart equipment to do surgery (and revival).
- Will save the hassle of people wanting to do a brig doc gimmick.

Their job will mostly be to manage Brig's health. Heal damaged prisoners, heal that one guy who got harmbatoned on his arrest, take care of the officer who just got robusted by permas, cure officers dying by viruses, check for implants on traitors, etc.

**Comment by ATH1909** [comment]
you might want to add a defib or two to the brig's medical section; not all medical players know that the revival surgery exists

File: fulp_modules/jobs/brigdoc/code/brigdoc.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	exp_type = EXP_TYPE_CREW
+	exp_type_department = EXP_TYPE_SECURITY
+	id_icon = 'fulp_modules/jobs/cards.dmi'
+	hud_icon = 'fulp_modules/jobs/huds.dmi'
+	fulp_spawn = /obj/effect/landmark/start/brigdoc
+
+	outfit = /datum/outfit/job/brigdoc
+
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_PHARMACY, ACCESS_CHEMISTRY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
+	minimal_access = list(ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
```
Comment (ATH1909) [inline fulp_modules/jobs/brigdoc/code/brigdoc.dm:21]: do they have the access required to access the surgery tools in the brig on all maps?

Comment (SgtHunk) [reply fulp_modules/jobs/brigdoc/code/brigdoc.dm:21]: They get a full med belt with surgery tools

**Comment by SgtHunk** [comment]
> you might want to add a defib or two to the brig's medical section; not all medical players know that the revival surgery exists

That would mess with TG map files unless it simply was on their hand, revival surgery is pretty easy to do and for a role that will mainly be focused on surgery and treatment on the brig, it should be common knowledge.

File: config/jobs.txt [hunk 2]
```diff
@@ -44,6 +44,7 @@ Roboticist=2,2
 Warden=1,1
 Detective=1,1
 Security Officer=5,5
+Brig Physician=2,1 //FULP
```
[the remark this answers is not in the document]
Comment (SgtHunk) [reply config/jobs.txt:47]: Chances are, not really
this was mainly for testing latejoins
And well, it's config so tweaking it shouldnt be hard on the repo's end

**Comment by remove32** [comment]
alright well I just TM'd this and it works fine. not surprising as the concept is low risk

**Comment by remove32** [comment]
I am somewhat concerned that this role might get used to powergame. Having direct access to both Medbay and Security roundstart is a dream for people like that

**Comment by SgtHunk** [comment]
> I am somewhat concerned that this role might get used to powergame. Having direct access to both Medbay and Security roundstart is a dream for people like that

The brigdoc gimmick is seen semi-commonly on Fulp, and afaik it doesn't result in a lot of powergaming, just in a brig with less of a mess in the form of death and blood
They have slightly more access than a paramedic medbay-wise on normal pop
That is, just basic medbay access + mech medical + surgery room + morgue.
On lowpop, they also get access to pharmacy (which MDs on normal pop get)
On the brig, they only get 'security' (office iirc) access on lowpop.
On normal pop, they only get sec doors + brig itself.

**Comment by remove32** [comment]
alrighty. we'll just have to keep eye on it

**Comment by ATH1909** [comment]
> I am somewhat concerned that this role might get used to powergame. Having direct access to both Medbay and Security roundstart is a dream for people like that

sec officers that have been assigned to the medbay department already get sec+medbay access

**Pull request merged**